diff --git a/gefyra/local/telemetry.py b/gefyra/local/telemetry.py
index 25fdee9..c8932f3 100644
--- a/gefyra/local/telemetry.py
+++ b/gefyra/local/telemetry.py
@@ -3,7 +3,6 @@ import configparser
 import uuid
 from pathlib import Path
 
-from cli_tracker.sdk import CliTracker
 from gefyra.configuration import __VERSION__
 
 logger = logging.getLogger("gefyra")
@@ -48,12 +47,7 @@ class CliTelemetry:
             self._init_tracker(user_id=user_id)
 
     def _init_tracker(self, user_id):
-        self.tracker = CliTracker(
-            application="gefyra",
-            dsn=SENTRY_DSN,
-            release=__VERSION__,
-            fingerprint=user_id,
-        )
+        pass
 
     def _get_user_id(self, config):
         if config["telemetry"].get("id"):
diff --git a/pyproject.toml b/pyproject.toml
index 80e8d0a..58711d5 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,7 +24,6 @@ classifiers = [
 python = "^3.8"
 kubernetes = ">=24.2,<26.0"
 docker = "^6.0.0"
-cli-tracker = ">=0.2.7,<0.4.0"
 tabulate = ">=0.8.10,<0.10.0"
 
 [tool.poetry.dev-dependencies]
